_delphi_utils_python/delphi_utils/export.py

```diff
@@ -2,16 +2,19 @@
 # -*- coding: utf-8 -*-
 from datetime import datetime
 from os.path import join
+from typing import Optional
 
+import numpy as np
 import pandas as pd
 
 def create_export_csv(
     df: pd.DataFrame,
-    start_date: datetime,
     export_dir: str,
-    metric: str,
     geo_res: str,
     sensor: str,
+    metric: Optional[str] = None,
+    start_date: Optional[datetime] = None,
+    end_date: Optional[datetime] = None
```

Comment:
just confirming for future forensics: i've done a search, and none of the indicators that currently call this function use positional arguments to specify `start_date` or `metric`, so the change in signature here isn't going to break anything